tests/monitoring/monitoring.go

```diff
@@ -160,10 +167,13 @@ var _ = Describe("[Serial][sig-monitoring]Monitoring", Serial, decorators.SigMon
 			kv, err := virtClient.KubeVirt(originalKv.Namespace).Get(context.Background(), originalKv.Name, metav1.GetOptions{})
 			Expect(err).ToNot(HaveOccurred())
 
-			kv.Spec.CustomizeComponents = v1.CustomizeComponents{}
+			patchData := map[string]interface{}{}
+
+			patchBytes, err := json.Marshal(patchData)
+			Expect(err).ToNot(HaveOccurred())
```

Comment:
Please use json patch; e.g.
```suggestion
            patchBytes, err := patch.New(WithRemove("/spec/customizeComponents").GeneratePayload()
			Expect(err).ToNot(HaveOccurred())
			patchData := map[string]interface{}{}
```